core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1063,6 +1063,14 @@ public enum Property {
       "1.3.5"),
   TABLE_ARBITRARY_PROP_PREFIX("table.custom.", null, PropertyType.PREFIX,
       "Prefix to be used for user defined arbitrary properties.", "1.7.0"),
+  TABLE_COMPACTION_INPUT_DROP_CACHE_BEHIND("table.compaction.input.drop.cache", "*",
+      PropertyType.STRING,
+      "Sets FSDataInputStream.setDropBehind(true) is set on compaction"
+          + " input streams for the specified type of files. The value of this property is a comma-separated list of Accumulo"
+          + " file type prefixes ('A', 'C', 'F', 'I', and 'M'). The value '*' can be used to"
+          + " specify all file types (default). When the input file prefix matches a value in this list,"
```

Comment:
I don't think these are documented for users anywhere. I have reservations about making the file naming conventions part of the user contract, since it's so tightly coupled to the internals. But, I don't have a better idea for how to add this kind of granularity... except maybe we enumerate a kind of "TYPE" here, rather than a filename convention that we use internally.

Reply:
Are you thinking that we should change the expected values from their prefix to their name (e.g. `BULK_IMPORT` vs `I`)?

Reply:
I put up a proposed replacement in #5485